### PR TITLE
Fixed some mistaked in the graph structure

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

**Comment by jsight** [comment]
Will be part of https://github.com/windup/windup/pull/926

**Comment by OndraZizka** [comment]
retest

**Comment by OndraZizka** [comment]
[ERROR] /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-rulesets/src/test/java/org/jboss/windup/rules/tests/WindupRulesTest.java:[309,18] cannot find symbol
  symbol:   method setProjectModel(org.jboss.windup.graph.model.ProjectModel)
  location: variable inputPath of type org.jboss.windup.graph.model.resource.FileModel

**Comment by jsight** [comment]
Yeah, there is a related products on rulesets that is required along with
this.

File: graph/api/src/main/java/org/jboss/windup/graph/model/ArchiveModel.java [hunk 1]
```diff
@@ -43,56 +49,51 @@
...
-     */
-    @Adjacency(label = "childArchive", direction = Direction.OUT)
-    Iterable<ArchiveModel> getChildrenArchive();
-
-    /**
-     * Contains the children of this archive.
-     */
-    @Adjacency(label = "childArchive", direction = Direction.OUT)
-    void addChildArchive(final ArchiveModel resource);
-
```
Comment (OndraZizka) [inline graph/api/src/main/java/org/jboss/windup/graph/model/ArchiveModel.java]: These disappeared without a replacement?

Comment (jsight) [inline graph/api/src/main/java/org/jboss/windup/graph/model/ArchiveModel.java]: Yes, they don't appear to be used anywhere.

**Pull request merged**